config/database.yml

```diff
@@ -10,11 +10,11 @@ default: &default
 
 development:
  <<: *default
- database: sample_project_development
+ database: target_development
 
 test:
  <<: *default
- database: sample_project_test
+ database: target_test
 
 production:
  <<: *default
```

Comment:
We will need to rebase the branch given that we are ignoring this file on #2 I believe